index.js

```diff
@@ -65,9 +65,15 @@ function onFinished (msg, listener) {
 function isFinished (msg) {
   var socket = msg.socket
 
+  if (msg.stream) {
+    // Http2ServerRequest
+    // Http2ServerResponse
+    return msg.stream.closed
```

Comment:
Not this is needed in order to not break http2 compat which does not implement `outputSize`.